src/utils/components/Labels/OSLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Label } from '@patternfly/react-core';
+
+import './os-labels.scss';
+
+type OSLabelProps = {
+  text: string;
+  icon: ReactNode;
+};
+const OSLabel: FC<OSLabelProps> = ({ text, icon }) => {
+  return (
```

Comment:
You can omit the `return` in here, if there is nothing else in the component, except the return function...